.env

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)